database/src/main/kotlin/no/nav/su/se/bakover/database/søknadsbehandling/SøknadsbehandlingPostgresRepo.kt

```diff
@@ -503,7 +503,7 @@ internal class SøknadsbehandlingPostgresRepo(
         val attesteringer = string("attestering").toAttesteringshistorikk()
         val søknadsbehandlingHistorikk =
             SøknadsbehandlingshistorikkJson.toSøknadsbehandlingsHistorikk(string("saksbehandling"))
-        val saksbehandler = NavIdentBruker.Saksbehandler(string("saksbehandler"))
+        val saksbehandler = stringOrNull("saksbehandler")?.let { NavIdentBruker.Saksbehandler(it) }
```

Comment:
Å dette er noe vi ikke kan catche før db laget eller er det for eksisterende søknader uten behandling?
Virker for meg som om det er lagt opp til flere steg her for å få dette inn uten at det kommer tydelig frem i pren så blir litt gjetting på feedbacken her.